src/tests/class_tests/openms/data/MRMAssay_reannotateTransitions_output_1.TraML

```diff
@@ -25,7 +25,7 @@
     </Protein>
   </ProteinList>
   <CompoundList>
-    <Peptide id="1_sp|O75882|ATRN_HUMAN_AAAAAAVSGSAAAEAK_2_50.01" sequence="AAAAAAVSGSAAAEAK">
```

Comment:
Peptides and transitions are ordered differently. The peptide and transition indices are taken from the input and not generated from scratch.